pkg/gcp/actuator/actuator.go

```diff
@@ -455,6 +455,17 @@ func (a *Actuator) needsUpdate(ctx context.Context, cr *minterv1.CredentialsRequ
 	if gcpStatus.ServiceAccountID != "" {
 		// serviceAccountID non-"" means we're in mint-mode
 
+		_, err := getServiceAccount(readClient, gcpStatus.ServiceAccountID)
+		logger.WithField("serviceaccount", gcpStatus.ServiceAccountID).Infof("looool, %v", err)
```

Comment:
extra debugging that forgot to get removed?

Reply:
LOL yes